synapse/media/media_repository.py

```diff
@@ -214,7 +214,10 @@ async def create_content(
             user_id=auth_user,
         )
 
-        await self._generate_thumbnails(None, media_id, media_id, media_type)
+        try:
+            await self._generate_thumbnails(None, media_id, media_id, media_type)
+        except Exception as e:
+            logger.info("Failed to generate thumbnails: %s", e)
```

Comment:
Maybe a warning? It is tough though since it is user content, what's an admin to do?

I think catching the error and having the image uploaded even if thumbnails fail is the correct thing to do though!